README.md

```diff
@@ -180,7 +180,7 @@ Have dataset(s) in one of the following format (JSONL recommended):
   ```json
   {"instruction": "...", "input": "...", "output": "..."}
   ```
-- `sharegpt:chat`: conversations where `from` is `human`/`gpt`
+- `sharegpt`: conversations where `from` is `human`/`gpt`
```

Comment:
We need to ensure smooth transition. There are a lot of configs with the old names.

I think we should for now in config auto convert the old names to the new ones and log a deprecation warning.